### PR TITLE
feat: adds writing access key + secret key to ssm pstore

### PR DESCRIPTION
**Pull request by Gowiem**
## what
* Adds writing the created access key ID and secret access key to SSM Parameter Store

## why
* There is no great way of getting these values out of this module outside of utilizing them in an `output` block in the root module. This is inconvenient and I believe it's preferable to lookup these values from parameter store instead of writing them to `output` (even as sensitive) all the time. 

## references
* N/A

### CODE REVIEW

**Comment by nitrocode** [comment]
It would be nice to be able to disable the creation of the ssm params

**Comment by Gowiem** [comment]
@nitrocode Good idea -- I just added a flag for that.

File: variables.tf [hunk 1]
```diff
@@ -39,3 +39,9 @@ variable "create_iam_access_key" {
   description = "Whether or not to create IAM access keys"
   default     = true
 }
+
+variable "write_keys_to_ssm_enabled" {
```
Comment (nitrocode) [inline variables.tf:43]: Why not simply this ?
```suggestion
variable "ssm_enabled" {
```

**Review by nitrocode — APPROVED** [review]

File: main.tf [hunk 2]
```diff
@@ -42,6 +46,32 @@ resource "aws_iam_user_policy_attachment" "policies" {
...
+
+module "store_write" {
+  source  = "cloudposse/ssm-parameter-store/aws"
+  version = "0.8.0"
+
+  count = module.this.enabled && var.ssm_enabled ? 1 : 0
+
+  parameter_write = [
+    {
+      name        = "/system_user/${local.username}/access_key_id"
```
Comment (nitrocode) [inline main.tf:61]: Hmm usually we expose this prefix too but we can always do that in a separate pr
